apps/financial-aid/backend/migrations/20220322130235-staff-remove-muncipality_homepage.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.removeColumn('staff', 'municipality_homepage', { transaction: t }),
+      ]),
+    )
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.addColumn('staff', 'municipality_homepage', {
```

Comment:
Do we not need to specify the column type and nullable here?
Similar to this: 
![image](https://user-images.githubusercontent.com/43727721/159491551-a9232e95-7def-4bb7-a097-b5c33ab9e649.png)

Reply:
Well spotted 👁️ 👁️